gistUpload.js

```diff
@@ -1,27 +1,34 @@
 const axios = require('axios');
 const fileSystem = require('fs');
+require('dotenv').config();
 
 const API_URL = 'https://api.github.com/gists';
 let finalString = '';
 const PROG_BAR = '=';
 const BAR_LENGTH = 30;
 const HANG_PERCENT = 0.25;
 
+
 /** This is a description of the uploadFile function
  * @param {string} contents - Contents of file from readFile
  * @param {string} gist - Name of gist supplied by user
  */
 function uploadFile(contents, gist) {
   const post = { [gist]: { content: contents } };
-  return axios.post(
-    API_URL,
-    { files: post },
-  )
+  return axios({
+    method: 'post',
+    url: API_URL,
+    auth: {
+      user: 'arianna-h-h',
```

Comment:
Shouldn't this come from `process.env` ?

Reply:
done